src/steps/big-table/converters.ts

```diff
@@ -0,0 +1,222 @@
+    entityData: {
+      source: operation,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_OPERATION,
+        _type: ENTITY_TYPE_BIG_TABLE_OPERATION,
+        _key: getOperationKey(operation),
+        name: operation.name,
+        done: operation.done,
+      },
+    },
+  });
+}
+
+export function createInstanceEntity({
+  instance,
+  projectId,
+}: {
+  instance: bigtableadmin_v2.Schema$Instance;
+  projectId: string | undefined | null;
+}) {
+  const instanceName = getLastUrlPart(instance.name!);
+
+  return createGoogleCloudIntegrationEntity(instance, {
+    entityData: {
+      source: instance,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_INSTANCE,
+        _type: ENTITY_TYPE_BIG_TABLE_INSTANCE,
+        _key: getInstanceKey(instance),
+        name: instance.name,
+        displayName: instance.displayName!,
+        active: instance.state === 'READY',
+        state: instance.state,
+        type: instance.type,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceName}/overview?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
+
+export function createAppProfileEntity({
+  appProfile,
+  projectId,
+  instanceId,
+}: {
+  appProfile: bigtableadmin_v2.Schema$AppProfile;
+  projectId: string | undefined | null;
+  instanceId: string | undefined | null;
+}) {
+  const instanceName = getLastUrlPart(instanceId!);
+  const appProfileName = getLastUrlPart(appProfile.name!);
+
+  return createGoogleCloudIntegrationEntity(appProfile, {
+    entityData: {
+      source: appProfile,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_APP_PROFILE,
+        _type: ENTITY_TYPE_BIG_TABLE_APP_PROFILE,
+        _key: getAppProfileKey(appProfile),
+        name: appProfile.name,
+        instanceId,
+        etag: appProfile.etag,
+        description: appProfile.description,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceName}/app-profiles/${appProfileName}?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
+
+export function createClusterEntity({
+  cluster,
+  projectId,
+  instanceId,
+}: {
+  cluster: bigtableadmin_v2.Schema$Cluster;
+  projectId: string | undefined | null;
+  instanceId: string | undefined | null;
+}) {
+  return createGoogleCloudIntegrationEntity(cluster, {
+    entityData: {
+      source: cluster,
+      assign: {
+        _class: ENTITY_CLASS_BIG_TABLE_CLUSTER,
+        _type: ENTITY_TYPE_BIG_TABLE_CLUSTER,
+        _key: getClusterKey(cluster),
+        name: cluster.name,
+        instanceId,
+        state: cluster.state,
+        serveNodes: cluster.serveNodes,
+        defaultStorageType: cluster.defaultStorageType,
+        location: cluster.location,
+        kmsKeyName: cluster.encryptionConfig?.kmsKeyName,
+        webLink: getGoogleCloudConsoleWebLink(
+          `/bigtable/instances/${instanceId}/overview?project=${projectId}`,
+        ),
+      },
+    },
+  });
+}
```

Comment:
1) You can also add that `active` thing here :slightly_smiling_face: 

2) I'm super glad you added `kmsKeyName` field to this :+1:. Also, whenever you encounter an entity that has `kmsKeyName` field in it somewhere we'd want and build a relationship between the `kmsKeyName` if that option is used for said resources. Take a look at this for example: https://github.com/Creativice-Oy/graph-google-cloud/blob/master/src/steps/big-query/index.ts#L62.

My suggestion - create a BigTable instance manually (or modify Terraform) so that it uses existing kms key (it needs to be in the same zone where the key resides iirc). Then just apply that modification - make sure to make that step depends on the step that ingests kms keys and update tests accordingly please.

Otherwise everything is great here!

Reply:
<img width="617" alt="Screenshot 2021-08-12 at 8 59 42 PM" src="https://user-images.githubusercontent.com/39617427/129232971-90e9d423-d5fd-4503-b995-7665079766ca.png">

Looked everywhere, but not being able to link a table to a kms key. :( 

There, indeed, is a kms key in the same region/zone as the table. The tests are passing too, but that probably is a false positive.